packages/convai-widget-core/src/widget/Sheet.tsx

```diff
@@ -91,52 +96,63 @@ export function Sheet({ open }: SheetProps) {
               : "bottom-20"
         )}
       >
-        <div className="bg-base shrink-0 flex gap-2 p-4 items-start">
-          <div className="relative w-16 h-16" />
-          <InOutTransition active={showTranscript && !isDisconnected.value}>
-            <StatusLabel className="rounded-bl-[calc(var(--el-bubble-radius)/3)] transition-opacity data-hidden:opacity-0" />
-          </InOutTransition>
-        </div>
-        <Transcript
-          transcript={filteredTranscript}
-          scrollPinned={scrollPinned}
-        />
-        <SheetActions
-          scrollPinned={scrollPinned}
-          showTranscript={showTranscript}
+        <SheetHeader
+          showBackButton={currentConfig.value.showHeaderBack}
+          onBackClick={currentConfig.value.onHeaderBack}
+          showStatusLabel={showTranscript && !isDisconnected.value}
+          showShadow={showTranscript}
+          showLanguageSelector={
+            currentContent.value !== "feedback" &&
+            (!showTranscript || isDisconnected.value)
+          }
         />
-        <InOutTransition active={!showTranscript || isDisconnected.value}>
-          <div className="absolute top-0 left-0 right-0 p-4 flex justify-center transition-[opacity,transform] duration-200 data-hidden:opacity-0 data-hidden:-translate-y-4">
-            <SheetLanguageSelect />
+        {currentContent.value === "transcript" && (
+          <>
+            <Transcript
+              transcript={filteredTranscript}
+              scrollPinned={scrollPinned}
+            />
+            <SheetActions
+              scrollPinned={scrollPinned}
+              showTranscript={showTranscript}
+            />
+          </>
+        )}
```

Comment:
Let's use `InOutTransition` to fade this in out:
```suggestion
        <InOutTransition active={currentContent.value === "transcript"}>
          <div className="grow flex flex-col min-h-0 transition-opacity duration-300 ease-out data-hidden:opacity-0">
            <Transcript
              transcript={filteredTranscript}
              scrollPinned={scrollPinned}
            />
            <SheetActions
              scrollPinned={scrollPinned}
              showTranscript={showTranscript}
            />
          </div>
        </InOutTransition>
```